app/views/swagger_ui/_swagger_ui.html.erb

```diff
@@ -10,7 +10,7 @@
         apiKey: '<%= api_key %>',
       <% end %>
       <% if defined?(validator_url) %>
-        validatorUrl: <%= validator_url %>,
+        validatorUrl: '<%= validator_url %>',
```

Comment:
https://github.com/swagger-api/swagger-ui#parameters 
Setting it to *null* will disable validation